docs/source/conf.py

```diff
@@ -14,7 +14,7 @@
 # import sys
 # sys.path.insert(0, os.path.abspath('.'))
 
-__version__ = "0.10.0-rc1"
```

Comment:
This probably isn't intentional, right?

Reply:
it's not, I didn't notice the tag in git